src/test/regress/sql/function_with_case_when.sql

```diff
@@ -0,0 +1,22 @@
+-- create function
```

Comment:
To avoid leaking objects from one test to another we generally create a dedicated schema for the test and set the search_path accordingly. The easiest way to do that is by running this script: https://github.com/citusdata/citus/blob/main/src/test/regress/bin/create_test.py